sdk/communication/Azure.Communication.Chat/src/Models/ChatMessageReadReceipt.cs

```diff
@@ -1,26 +1,37 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// <auto-generated/>
+
+#nullable disable
```

Comment:
instead of disabling in this file, we should should disable in the entire csproj file. This line should be removed: https://github.com/Azure/azure-sdk-for-net/blob/gelli/communicationIdentifierChange/sdk/communication/Azure.Communication.Chat/src/Azure.Communication.Chat.csproj#L14

Reply:
Sounds good, lets tackle it with the next set of changes that will address the azure sdk review comments. 
Would like to merge this branch to master today / tomorrow if we can.